src/en/help-azure-advanced.md

```diff
@@ -0,0 +1,172 @@
+Title: Using Microsoft Azure with Juju - advanced
+
+# Using Microsoft Azure with Juju - advanced
+
+This page is dedicated to more advanced topics related to using Microsoft Azure
+with Juju. The main page is [here][clouds-azure].
+
+## Manually adding Azure credentials
+
+The manual option is useful if Juju fails to automatically gather your
+credentials, or if you want to automate the process.
+
+### Gathering values
+
+We will need values for the following bits of information:
+
+ - application-id
+ - subscription-id
+ - application-password
+ - application-name
+ - tenant-id
+
+In the sections below, we will assign each of these a variable name.  When you
+enter them into the command, replace the variable name we give with the actual
+ID that corresponds to the variable.
+
+!!! Important:
+    This process requires the Azure CLI tool to be installed and a successful
```

Comment:
*and a successful login with it* sounds a little too minimal to me (ie. to be installed and used to successfully login to Azure prior to Juju configuration).